src/darker/tests/test_command_line.py

```diff
@@ -256,20 +256,6 @@ def test_parse_command_line(
     dict(config={"check": True}, expect_warn=set()),
     dict(config={"isort": True}, expect_warn=set()),
     dict(config={"lint": ["pylint"]}, expect_warn=set()),
-    dict(
-        config={"skip_string_normalization": True},
-        expect_warn={
-            "The configuration option `skip_string_normalization` in [tool.darker] is"
-            " deprecated and will be removed in Darker 3.0."
```

Comment:
I would love for the information that this is moved to `[tool.black]` now to be included here, so I don't have to research that when getting the warning.

Reply:
@dwt that would have indeed been a good thing during the deprecation period.

But what now when we're actually removing these opions? Are you saying it would still be good to have Darker recognize them but give a hard error mentioning `[tool.black]`, and stop?

Reply:
@dwt I now changed it so these configuration options are parsed from the configuration file, but a descriptive error is raised if they are used. The error instructs to use the `[tool.black]` section instead.

Reply:
I really like that. Thanks